iac-extensions/spring-config/src/main/antlr/PropertiesLexer.g4

```diff
@@ -7,11 +7,12 @@ public PropertiesLexer(CharStream input, boolean crLexerCostructor) {
 }
 }
 
-COMMENT   : [!#] -> pushMode(VALUE_MODE);
-NEWLINE   : [\r\n\u2028\u2029]+;
-DELIMITER : [ ]* [:=\t\f ] [ ]* -> pushMode(VALUE_MODE);
-SLASH     : '\\' -> more, pushMode(INSIDE);
-CHARACTER : ~ [:=\r\n\u2028\u2029];
+COMMENT               : [!#] -> pushMode(VALUE_MODE);
+START_OF_FILE_SPACING : {this.getCharPositionInLine() == 0}? [ \t\f\r\n\u2028\u2029]+ -> channel(HIDDEN);
```

Comment:
Actually, it is not a spacing at the beginning of the file, but at the beginning of the line. So `LEADING_SPACE` or `SPACE` or `WHITESPACE` will be a better name.